components/gfx/text/glyph.rs

```diff
@@ -547,6 +547,25 @@ impl<'a> GlyphStore {
     }
 
     #[inline]
+    pub fn range_index_of_advance(&self, range: &Range<ByteIndex>, advance: Au, extra_word_spacing: Au) -> (usize, Au) {
```

Comment:
While describing what this function returns I realized that it's more intuitive if it returns `(index, current_advance)` instead of `(index, advance -current_advance)`. This just needed a slight modification in the part that invokes this.
I'll add some comments after I push the modifications.